compredict/client.py

```diff
@@ -23,11 +24,15 @@
 class api:
 
     def __init__(self,
+                 username: Optional[str] = None,
+                 password: Optional[str] = None,
                  token: Optional[str] = None,
+                 token_refresh: Optional[str] = None,
```

Comment:
I would rename it to `refresh_token` as it is the standard name in Auth 2.0

Reply:
I think I named it this way so the class is not confusing it with method: refresh_token(). But you are right, I will just rename method instead.